### ISSUE TITLE
Bliss version number is not updated in the UI after upgrade

### ISSUE THREAD

**Issue by seangordon** [seangordon/QNAP-bliss#3]
The default QPKG configuration relies on the version number being embedded in the QPKG.
I'm assuming that there is a way to update the version number shown in the QNAP UI when an upgrade happens, since other packages are doing this, but I have yet to find the relevant documentation in the QDK to show how this is done.

**Comment by gravelld** [comment]
There's also the question of finding what version is running of bliss. One way is to monitor stdout of the bliss process and look for "bliss version: ([0-9]{8})"

**Comment by crieke** [comment]
I was thinking about the same thing, when I created a qpkg for RoonServer.
I decided against displaying the installed application version number. Instead I showed the version of the package. The version of the installed binaries can be checked in the software itself. If this version number will be used to display it in the App Center, it will be hard to keep track of dedicated qpkg script files (if there is the need for a change in the future).
